k8s/Production-GCP/initiatives-webapi-service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: initiatives-webapi
+  namespace: production
+  labels:
+    app: initiatives-webapi
+    technology: dotnetcore
+    tier: frontend
+    product: octava
+spec:
+  type: NodePort
```

Comment:
I think this should be of type "ClusterIP"...

Reply:
GKE requires NodePort.